setup/pi/setup-teslausb

```diff
@@ -455,6 +461,7 @@ else
 fi
 
 update_package_index
+install_python_packages
```

Comment:
can this be made conditional on using sns? E.g. by moving the call to install_python_packages into configure_sns ?

Reply:
I see that configure.sh step can be skipped, so I propose to add a condition that SNS is configured directly in setup-teslausb to install the python packages. What do you think?

Reply:
configure.sh is only skipped when archiving is not configured, in which case no notifications will be sent anyway. If SNS needs python3/pip/boto3, it seems cleanest to install those in the script that configures SNS.